lib/gems/pending/appliance_console/external_auth_options.rb

```diff
@@ -42,7 +42,7 @@ def ask_questions
         say("#{apply}) Apply updates")
         say("#{skip}) Skip updates")
         show_updates
-        selection = ask_for_integer("Choose option 1-#{skip}", 1..skip)
+        selection = ask_for_integer("Choose option for updating external authentication ", 1..skip, prompt_prefix: "")
```

Comment:
So, instead of changing the signature of `ask_for_integer` we could just change this string to "option to apply".